apps/molgenis-components/src/components/constants.ts

```diff
@@ -3,6 +3,8 @@ export default {
   CODE_9: 57,
   CODE_PERIOD: 46,
   CODE_MINUS: 45,
+  TABLE_NAME_REGEX: /^(?!.* _|.*_ )[a-zA-Z][a-zA-Z0-9 _]{0,30}$/,
+  COLUMN_NAME_REGEX: /^(?!.* _|.*_ )[a-zA-Z][a-zA-Z0-9 _]*$/,
```

Comment:
I think there is a maximum length here as well, at least dictated by postgresql (maybe 64?)

Reply:
It seems to be 63 (`length NAMEDATALEN - 1`) [1,2,3]. Will change accordingly.


Sources:
[1] https://www.postgresql.org/docs/current/runtime-config-preset.html#GUC-MAX-IDENTIFIER-LENGTH
[2] https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS
[3] running `psql -U molgenis -c "SHOW max_identifier_length;"` locally